### DISCUSSION TITLE
Week 9 post

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#188] [General]
I'm not sure if I ever officially posted for week 9. I was having challenges getting started on the project. I did post a lot of questions. Thanks to everyone who has been really helpful in that process. I'm still working on getting started, so hopefully I'll be able to add my code in later this week. @kensnyder I'll probably come to office hour tomorrow morning because I'm still having challenges.

I did download Insomnia, but I'm lost on what API's are. Everything from week 9 was a bit over my head. I was able to get the template set up last week, but I still need to add it all into the codespace. My task list is the same as previously posted.

**Comment by kensnyder** [comment]
@Skye404 Let me know if you need a one-on-one Zoom meeting.

**Reply by Skye404** [reply]
@kensnyder I think I do need to meet. I'm still having issues figuring out how to see the results of my code, but I think it's in the right place now. I'll message you in Canvas.
